test/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.test.cpp

```diff
@@ -542,7 +558,12 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator, Orekit
     expectedCoordinates << -5172890.413848375000, -4716058.194055955000, 0000000.000000000000, 05083.946625786208,
         -05576.415230079790, 00000.000000000000;
 
-    VectorXd residuals = finalState.getCoordinates() - expectedCoordinates;
+    VectorXd actualCoordinates(6);
```

Comment:
We should double check no other tests are doing `state.getCoordinates()`

Reply:
I confirmed this is good.